src/core/MeshBVH.js

```diff
@@ -595,20 +595,22 @@ export class MeshBVH {
 			intersectsTriangles,
 		} = callbacks;
 
-		const geometry = otherBvh.geometry;
-		const indexAttr = geometry.index;
-		const positionAttr = geometry.attributes.position;
+		const indexAttr = this.geometry.index;
+		const positionAttr = this.geometry.attributes.position;
+
+		const otherIndexAttr = otherBvh.geometry.index;
+		const otherPositionAttr = otherBvh.geometry.attributes.position;
 
-		tempMatrix.copy( matrixToLocal ).invert();
```

Comment:
Why did this line get removed? It's used below when running shapecast on line 670 to transform the geometry AABB into the local space of the other BVH.

Reply:
Simple mistake.